package.json

```diff
@@ -13,7 +13,7 @@
     "etc/prepare.js"
   ],
   "types": "bson.d.ts",
-  "version": "4.7.0",
+  "version": "5.0.0-alpha.2",
```

Comment:
We need to revert package.json and package-lock changes too.

Reply:
We'll probably want to do an alpha.3 and get the driver PR settled first so might as well keep this so the next bump is correct